code/src/OSFramework/OSStructure/DataChanges.ts

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSStructure {
+    export class DataChanges {
+        public changedRows: any;
```

Comment:
Please the change the type of `changedRows` to `Array` or `unknown`.

Reply:
fixed